### DISCUSSION TITLE
Does networkx have some of the shortest path algorithms like ALT that are more recent in the research community?

### DISCUSSION THREAD

**Discussion by jayavanth** [networkx/networkx#7566] [Algorithms]
I see that networkx has some APIs for Dijkstra's and A* but I was wondering if there are newer algorithms like ALT that make it quicker to find the shortest path

Basically, I want to find the shortest path given a city/state map. I found one implementation in Go https://github.com/AdrS/shortestpath/ but I was wondering if it's integrated into networkx or osmnx

**Comment by rossbar** [comment]
There are currently no open PRs relating to ALT, but from a quick skim of [the paper](https://www.cs.princeton.edu/courses/archive/spr06/cos423/Handouts/GH05.pdf) it seems like it'd be a good fit. I'd double-check with the geospatial libraries (e.g. osmnx, as you mentioned) to see whether there is already an implementation since most of the motivating examples from the internet seem to be shortest path search on transportation networks.
